server/api/protobuf/okane.proto

```diff
@@ -4,13 +4,15 @@ package okanepb;
 
 option go_package = "/okanepb";
```

Comment:
how about using go_package option as full url of this repository?

```suggestion
option go_package = "github.com/NikhilSharma03/Okane/server/api/protobuf/okane;okanepb";
```
```

Reply:
I think that would be better. Updated ✅